packages/simulation/pitop/simulation/sprites.py

```diff
@@ -199,7 +252,7 @@ def create_components(self):
             step=self.slider_step,
             initial=0,
             handleRadius=int(SLIDER_HANDLE_RADIUS * self.scale),
-            handleColour=pygame.Color("#505050"),
+            handleColour=PitopColor.CHARCOAL.value,
             colour=pygame.Color("#DBDEE0"),
```

Comment:
we should use the enum for this one too, though I wish they didn't need the .value thing